nhs_reminder/settings.py

```diff
@@ -21,6 +21,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
+RUN_TW_TESTS = False
```

Comment:
I haven't actually checked if errors cost money yet - I should ;) 

I think when we have a solution for #2 we can re-enable by default because we'll be using their test infrastructure rather than production system, but currently we'll just be hitting their production env, generating error reports etc for no benefit.